source/CardGames.py

```diff
@@ -308,10 +306,10 @@ def highest_card(hand):
 
 
 class Poker:
-  def __init__(self, players: "list[Player]", dealer: Dealer):
+  def __init__(self, players: "list[Player]", dealer: Dealer, pot: Pot):
```

Comment:
I don't see any logic errors as it aligns with the task requirement with the appropriate Call function!